bears/css/PostCSSBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.settings.Setting import typed_list
+
+
+class PostCSSBear(Lint, LocalBear):
+    executable = "postcss"
+    gives_corrected = True
+
+    def run(self, filename, file, postcss_plugins: typed_list(str)=[]):
+        '''
+        This bear adds ``PostCSS`` utilities to analyze CSS code.
+
+        :param postcss_plugins: List of PostCSS plugins to use with
+                                this bear which can be defined in
+                                .coafile.
+        '''
+        if not postcss_plugins:
+            self.err("Please specify at least one plugin name.")
+            return []
+
+        for plugin in postcss_plugins:
```

Comment:
You need to do more analysis on the invalidplugins.
If an invalid plugin is given then you mentioned postcss gives an error message.
 - Is that error message in stdout or stderr ?
 - Is the "corrected file" in stdout or stderr ?
We need to catch that and make Lint [_print_errors()](https://github.com/coala-analyzer/coala/blob/master/coalib/bearlib/abstractions/Lint.py#L102) with it